convert2rhel/redhatrelease.py

```diff
@@ -49,35 +49,42 @@ def get_system_release_content():
         loggerinst.critical("%s\n%s file is essential for running this tool." % (err, filepath))
 
 
-class YumConf:
-    _yum_conf_path = "/etc/yum.conf"
+class PkgManagerConf:
+    """
+    Check if the config file of the systems package manager has been modified and if it has then
+    remove those changes before the conversion completes.
+    NOTE: The pkg manager config file path only needs to be set to yum.conf as there is a symlink between yum and dnf.
```

Comment:
```suggestion
    .. note::  
        The pkg manager config file path only needs to be set to yum.conf as there is a symlink between yum and dnf.
```
I would go with this format: https://github.com/oamg/convert2rhel/blob/main/convert2rhel/initialize.py#L62